include/slate/internal/Memory.hh

```diff
@@ -54,14 +54,20 @@ public:
     /// which can be host.
     size_t available(int device) const
     {
-        return free_blocks_.at(device).size();
+        if (device == HostNum)
+            return 0;
```

Comment:
Why is it 0 now for `HostNum`?

Reply:
Previously, we never inserted anything into `free_blocks_.at(HostNum)`, so it's size was always 0.  So, we get matching behavior.